mne/tests/test_event.py

```diff
@@ -154,7 +154,7 @@ def test_find_events():
 
     stim_channel = 'STI 014'
     stim_channel_idx = pick_channels(raw.info['ch_names'],
-                                     include=stim_channel)
+                                     include=[stim_channel])
```

Comment:
So I think that what was happening was that since it was checking a string, then it would still work if you passed a string that was one of the channel names. e.g. this is one of the checking lines:

```python
for k, name in enumerate(ch_names):
         if (len(include) == 0 or name in include) and name not in exclude:
```
So if exclude == 'chan_1', and one of the channels is 'chan_1', then it will properly exclude this channel. However, it will also exclude any channels called, for example, 'ch', 'a', 'n', 'an_1', and so on. If exclude == ['chan_1'], then it will only check for the full name.